src/sentry/metrics/dogstatsd.py

```diff
@@ -24,9 +24,10 @@ class DogStatsdMetricsBackend(MetricsBackend):
     def __init__(self, prefix: str | None = None, **kwargs: Any) -> None:
         # TODO(dcramer): it'd be nice if the initialize call wasn't a global
         self.tags = kwargs.pop("tags", None)
+        kwargs["disable_buffering"] = False
```

Comment:
Should this be `statsd_disable_buffering`? That parameter name is more [aligned with dogstats](https://github.com/DataDog/datadogpy/blob/7b721e331d4287acd5bf86a5e06cb2d97c4f2a6d/datadog/__init__.py#L41-L146)

Reply:
great catch, thank you.

Reply:
Wait, doesn't this effectively make it identical to what we have now? https://github.com/DataDog/datadogpy/blob/7b721e331d4287acd5bf86a5e06cb2d97c4f2a6d/datadog/__init__.py#L143

Reply:
my hypothesis is as follows: there could be period before the buffering is enabled where it tries to flush metrics synchronously. by doing this on initialization, maybe this occurs less often / not at all. additionally, ts only calling this function once instead of twice